src/main/java/org/springframework/data/couchbase/repository/support/SimpleCouchbaseRepository.java

```diff
@@ -190,7 +191,12 @@ public void deleteAll() {
 	}
 
 	private QueryScanConsistency buildQueryScanConsistency() {
-		QueryScanConsistency scanConsistency = QueryScanConsistency.NOT_BOUNDED;
+		QueryScanConsistency scanConsistency = null;
+
+		if (couchbaseOperations.getConsistency() != null) {
+			scanConsistency = couchbaseOperations.getConsistency();
+		}
+
```

Comment:
Only the first line is necessary.  The Reactive*OperationSupport.build*Options() will take consistency from the template (couchbaseOperations) if not otherwise set.